python/paddle/v2/fluid/__init__.py

```diff
@@ -16,12 +16,13 @@
 from param_attr import ParamAttr
 from data_feeder import DataFeeder
 from core import LoDTensor, CPUPlace, GPUPlace
+import clip
 
```

Comment:
I have a concern, we use a related path of `clip`, the commonly used name `clip`, `io` may have a conflict with other packages.

Reply:
Python can rename every module when import